docs/ab-testing/AB_TESTING.md

```diff
@@ -201,9 +201,13 @@ You can now create A/B testing-driven rulesets .
     {
       "id": "1",
       "name": "AB testing rule",
-      "linkedComponent": {
```

Comment:
open question: is it worth leaving the `linkedComponent` in the documentation and mention that it is deprecated and will be removed in v12? And adding an example with `linkedComponents` as well.

Reply:
We should inform somehow that 'linkedComponent' is deprecated and 'linkedComponents' should be used instead. 
I agree that just changing the examples is not enough, mostly for the users which have already used 'linkedComponent' will be misleading.